packages/weak-memoize/types/tests.ts

```diff
@@ -0,0 +1,36 @@
+import weakMemoize from '@emotion/weak-memoize'
+
+type Foo = { bar: 'xyz' }
+
+declare class Qwe {
+  answer: number
+}
+
+// $ExpectType Array<Foo>
+weakMemoize((arg: Foo) => [arg])({ bar: 'xyz' })
+
+// $ExpectType string[]
+weakMemoize((arg: string) => [arg])('foo')
```

Comment:
@Andarist this test is failing currently, it conflicts with the test on line 19 which expects a string arg to be an error.

What is the intended way for this to work then i can fix

Reply:
this line can just get removed